src/mmw/mmw/settings/layer_settings.py

```diff
@@ -90,12 +90,72 @@
     ],
     'coverage': [
         {
-            'display': 'National Land Cover Database',
+            'display': 'National Land Cover Database 2016',
+            'code': 'nlcd-2016-30m',
+            'css_class_prefix': 'nlcd-2016-30m nlcd',
```

Comment:
The CSS class prefix has a qualifier for this specific layer `nlcd-2016-30m` but also the general `nlcd` class, which is used to fill in the background colors for the NLCD categories here:

https://github.com/WikiWatershed/model-my-watershed/blob/c7a6fcb82a08bb37c24c5fda479af9e37d328505/src/mmw/sass/utils/_variables.scss#L56-L79